cmd/internal/build/build.go

```diff
@@ -59,6 +61,12 @@ func runCmd(_ *base.Command, args []string) {
 		cl.SetDebug(cl.DbgFlagAll)
 		cl.SetDisableRecover(true)
 	}
+	modload.LoadModFile()
```

Comment:
In most case, These three things are required together. They can merge to one function, eg. `modload.Load()`